dashboard/client/src/pages/node/NodeRow.tsx

```diff
@@ -127,6 +127,9 @@ export const NodeRow = ({
       <TableCell>
         <StatusChip type="node" status={raylet.state} />
       </TableCell>
+      <TableCell>
+        <Box minWidth={TEXT_COL_MIN_WIDTH}>{raylet.message}</Box>
```

Comment:
can you use the `CodeDialogButtonWithPreview` component so that there is a button to expand this and show more.

Reply:
Done.